jgrapht-core/src/test/java/org/jgrapht/alg/tour/NearestNeighborHeuristicTSPTest.java

```diff
@@ -0,0 +1,192 @@
+/*
+ * (C) Copyright 2020-2020, by Hannes Wellmann and Contributors.
+ *
+ * JGraphT : a free Java graph-theory library
+ *
+ * See the CONTRIBUTORS.md file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the
+ * GNU Lesser General Public License v2.1 or later
+ * which is available at
+ * http://www.gnu.org/licenses/old-licenses/lgpl-2.1-standalone.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR LGPL-2.1-or-later
+ */
+package org.jgrapht.alg.tour;
+
+import static java.util.stream.Collectors.*;
+import static org.hamcrest.MatcherAssert.*;
+import static org.hamcrest.Matchers.*;
+
+import org.apache.commons.math3.geometry.euclidean.twod.*;
+import org.jgrapht.*;
+import org.jgrapht.alg.interfaces.*;
+import org.jgrapht.generate.*;
+import org.jgrapht.graph.*;
+import org.junit.*;
+
+import java.io.*;
+import java.net.*;
+import java.util.*;
+
+public class NearestNeighborHeuristicTSPTest
```

Comment:
Usually, we add a comment stating that these are unit tests for the specified class and also authorship notice

Reply:
Thanks. I forgot that.